Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node as react-build
+COPY . /code
```

Comment:
Would it be possible to pre-fetch-and-cache some dep steps so that image layer caching would be used?  Otherwise each build takes more then four minutes for me even without changing anything in the source code...